client/my-sites/plans/jetpack-plans/more-info-box/index.tsx

```diff
@@ -1,35 +1,28 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { Button, Gridicon } from '@automattic/components';
 import * as React from 'react';
-import useTrackCallback from 'calypso/lib/jetpack/use-track-callback';
-import { PLAN_COMPARISON_PAGE } from 'calypso/my-sites/plans/jetpack-plans/constants';
 import type { TranslateResult } from 'i18n-calypso';
 import type { ReactNode } from 'react';
 
 import './style.scss';
 
 type MoreInfoProps = {
-	headline: TranslateResult;
 	buttonLabel: TranslateResult | ReactNode;
-	onButtonClick?: () => void;
+	buttonLink: string;
+	track: string;
```

Comment:
```suggestion
	trackEventName: string;
```

Maybe something like this would be more informative?

Reply:
Good call on this one, thanks!